scripts/form-utils.js

```diff
@@ -55,15 +55,17 @@ function disableFormFields(form) {
  * @param {HTMLFormElement} form
  */
 function enableFormFields(form) {
-    const formFields = form.querySelectorAll('input, button, select');
+    const formFields = form.querySelectorAll('input, button, select, textarea');
 
     for (let i = 0; i < formFields.length; i++) {
         formFields[i].disabled = false;
     }
 }
 
-function postToBasket(email, params, url, successCallback, errorCallback) {
+function postToEmailServer(params, successCallback, errorCallback) {
     const xhr = new XMLHttpRequest();
+    const url = "https://www-demo2.allizom.org/email-meico/";
```

Comment:
Suggestion: include locale (which will always be `en-US` for this view?) to make request a bit faster

```suggestion
    const url = "https://www-demo2.allizom.org/en-US/email-meico/";
```

Reply:
Good idea!